test/new_device_startmode_test.rb

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env rspec
```

Comment:
This test was completely removed in network-ng

Reply:
Yes, you are right. Dropped.